### ISSUE TITLE
Document how to complete a staging review for IA

### ISSUE THREAD

**Issue by erinrwhite** [department-of-veterans-affairs/va.gov-team#74552]
## User Story
As IA on the Governance Team, I want to have clear, up-to-date guidance on how to complete a Staging Review that maintains existing IA best practices from the soon-to-be-retired VA.gov Experience Standards 1.0.

Assignee: @erinrwhite 
Peer Reviewer: @shiragoodman 

## Description
As we update our VA.gov experience standards (#45160) to be more broad and focused on user experience, we are reducing the level of detail about some IA-specific requirements that are not currently covered in the design system. We are partially addressing this through #73149 and will continue work in #73684. This work will update internal documentation for the IA on the Governance Team to ensure internal documentation contains relevant details for an IA staging review.

## Impacted Artifacts

- [How to complete an IA review for Staging review touchpoint](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2302312688/How+to+complete+an+IA+review+for+Staging+review+touchpoint)

## Tasks
- [x] Update links to relevant and up-to-date VA guidance
- [x] Update "conduct your review" section to reflect current IA review workflow, incorporating relevant IA-specific heuristics from [VA.gov Experience Standards 1.0](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/va-gov-experience-standards)

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [x] "How to complete an IA review for Staging review touchpoint" is updated to reflect current processes

## Team Notification
- [x] Slack post on #platform-governance-team
- [ ] Presentation at Weekly Initiative Meeting
- [ ] No notification needed

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [ ] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [ ] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by erinrwhite** [comment]
Ticket begun but could use a review - thanks @shiragoodman

**Comment by shiragoodman** [comment]
@erinrwhite this looks great. Thank you! I updated the sprint and assigned it to the FY24Q2 tech debt epic.

One thing I'd recommend is linking the CAIA IA guidance you shared with me last week. I imagine that would be a helpful point of reference for all SR's moving forward.

**Comment by shiragoodman** [comment]
@erinrwhite thanks for the ping on Slack. I didn't get a Github notification due to the change in settings. All fixed now!

This looks GREAT! I looked at previous versions and we did not have as much detail. I like how you've broken it down into the different categories for your IA review - navigation, links, entry/exit points, etc. This is extremely helpful for me as a PM to know what's in scope for you. 

I really don't have much feedback, but the only thing I'd recommend is taking a look at the How to complete an accessibility review process. That doc includes a step for familiarizing oneself with the product. I would imagine the IA process should do the same. Looking at the content and design guidance docs, they may also need a 2nd look soon.

I would encourage you to share this doc with our Governance team on Slack and ask for the teams feedback. Is there anything they expect from you as the IA SME that's not included? Is there something they're doing as reviewers that you're not? When we drafted these docs, we did cross-team peer review, so I'd like the team to chime in as well.
